src/main/java/org/candlepin/subscriptions/capacity/admin/UsageContextSubscriptionProvider.java

```diff
@@ -121,14 +118,6 @@ public Optional<Subscription> getSubscription(
     }
 
     if (activeSubscriptions.size() > 1) {
-      // If this is a multipart ID, throw an error since correct result could not be determined.
-      if (billingAccountId.contains(";")) {
```

Comment:
No change required in this PR but we use this SUBSCRIPTION_CANNOT_BE_DETERMINED error in swatch-azure-producer in order to determine if we can retry processing.  I created a follow up task for this because I think it's out of scope for this card.
https://issues.redhat.com/browse/SWATCH-2595